drools/drools-core/src/main/java/org/drools/core/marshalling/impl/DocumentMarshallingStrategy.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright 2020 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.drools.core.marshalling.impl;
+
+import java.io.IOException;
+import java.io.ObjectInputStream;
+import java.io.ObjectOutputStream;
+
+import org.kie.api.marshalling.ObjectMarshallingStrategy;
+
+public class DocumentMarshallingStrategy implements ObjectMarshallingStrategy {
```

Comment:
Do we real need this class? Perhaps, could just provide the default implementation at ObjectMarshallingStrategy